src/v1/device_manager_client.ts

```diff
@@ -2527,23 +2526,40 @@ export class DeviceManagerClient {
       protos.google.cloud.iot.v1.IListDevicesResponse
     ]
   > | void {
-    request = request || {};
-    let options: CallOptions;
-    if (typeof optionsOrCallback === 'function' && callback === undefined) {
-      callback = optionsOrCallback;
-      options = {};
-    } else {
-      options = optionsOrCallback as CallOptions;
-    }
-    options = options || {};
-    options.otherArgs = options.otherArgs || {};
-    options.otherArgs.headers = options.otherArgs.headers || {};
-    options.otherArgs.headers['x-goog-request-params'] =
-      this._gaxModule.routingHeader.fromParams({
-        parent: request.parent || '',
+    return new Promise(async (resolve, reject) => {
+      let payload = JSON.stringify({
+        parent : request?.parent
       });
-    this.initialize();
-    return this.innerApiCalls.listDevices(request, options, callback);
+
+      var options = {
+        host: 'iot-sandbox.clearblade.com',
+        path: `/api/v/4/webhook/execute/`+ adminSystemKey +`/devices?parent=` + request?.parent,
```

Comment:
The webhook is called `cloudiot_devices` now, not `devices`. Also, this should use a system key for a template system, not the admin system key.

Reply:
Updated

Reply:
@clarkbynum